Gruntfile.js

```diff
@@ -181,9 +181,22 @@ module.exports = function (grunt) {
 	];
 
 	grunt.initConfig(gruntConfig);
-
 	grunt.registerTask('compiler', ['curl:compiler', 'unzip:compiler']);
 	grunt.registerTask('all', ['copy', 'concat', 'closure-compiler']);
+
+	grunt.loadTasks('spec/tasks');
+
+	var browsers = grunt.option('browsers');
+  grunt.registerTask('test',
```

Comment:
Your spelling looks borken too ;)  Looking in my editor it's fine, so must be a tab / space issue.